vuu-ui/packages/vuu-filters/src/column-filter/ColumnFilter.tsx

```diff
@@ -80,37 +102,48 @@ export const ColumnFilter = ({
               data-embedded
               sentiment="neutral"
             >
-              {operator}
+              {op}
             </Button>
           </MenuTrigger>
           <MenuPanel>
-            <MenuItem>=</MenuItem>
-            <MenuItem>!=</MenuItem>
-            <MenuItem>starts</MenuItem>
-            <MenuItem>ends</MenuItem>
-            <MenuItem>contains</MenuItem>
+            <MenuItem onClick={() => onOperatorChange("=")}>=</MenuItem>
```

Comment:
The menuitems for operator need to be built dynamically as they will be different, depending on data type of column. See the existing `OperatorPicker` component, which used getOperators. to do exactly this.